src/wallet/rpc/spend.cpp

```diff
@@ -439,8 +439,8 @@ RPCHelpMan settxfee()
         throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("txfee cannot be less than min relay tx fee (%s)", pwallet->chain().relayMinFee().ToString()));
     } else if (tx_fee_rate < pwallet->m_min_fee) {
         throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("txfee cannot be less than wallet min fee (%s)", pwallet->m_min_fee.ToString()));
-    } else if (tx_fee_rate > max_tx_fee_rate) {
-        throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("txfee cannot be more than wallet max tx fee (%s)", max_tx_fee_rate.ToString()));
+    } else if (tx_fee_rate > pwallet->m_max_tx_fee_rate) {
```

Comment:
in "[wallet]: add maxfeerate wallet startup option" (https://github.com/bitcoin/bitcoin/pull/29278/commits/6a2c7722698d22c95220daf7727f9219ed843042):

The old `max_tx_fee_rate` on L435 should be deleted: 

https://github.com/bitcoin/bitcoin/pull/29278/commits/6a2c7722698d22c95220daf7727f9219ed843042#diff-26141d9c7da21eeb4b9e3ffedfaad83212d4710a9e62888f7abea076ca1d0538R435

Reply:
Deleted 👍🏾